src/ast/mod.rs

```diff
@@ -476,14 +476,18 @@ pub enum Expr {
     },
 }
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub struct IntFormat {
+    pub unsigned: bool,
```

Comment:
Might prefer `signed` rather than `unsigned` to avoid double negatives

Reply:
I went with `unsigned` since it's more of an exception to the norm than anything. I tend to structure my bools with `true` as the "do something different" value so that they don't need to be negated whenever being read, but I guess that doesn't matter as much with `match` since you have to be explicit in both cases.

Reply:
`true` as "do something different" makes sense in C-likes where it's easy to zero-initialize things.  It doesn't make so much sense in rust.